freepbx/usr/share/phonebooks/samples/gamma_enterprise.php

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/php -q
+<?php
+/****************************
+*  Gamma DB credentials  *
+*****************************/
+$dsn="teamsystem";
+$user="sa";
+$pass="Teamsystem01";
+/****************************/
+
+$code = 0;
+
+// Get NethServer phonebook database credentials
+exec('perl -e \'use NethServer::Password; my $password = NethServer::Password::store(\'PhonebookDBPasswd\')  ; printf $password;\'',$out);
```

Comment:
Same as the other

Reply:
I've updated the business_nts sample and added a library to make it work. ref [this](https://mattermost.nethesis.it/nethesis/pl/9xfdpsmqzpf7u819ibkn9kmhmo) private chat